modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -93,13 +66,43 @@ This is the target message type and the Endpoint where you want to route the tra
 
 * xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-. After you create the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow), and click *Deploy* to deploy the configurations to your Mule Runtime.
+. After you create the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow, and click *Deploy* to deploy the configurations to your Mule Runtime.
 +
-Your message flow configuration is deployed to your Mule runtime engine and you can start receiving messages from your partner. Use the Activity page to track processed messages.
+. Your message flow configuration is deployed to your Mule runtime engine and you can start receiving messages from your partner.
 +
-Once a message flow is successfully deployed, you can track its processed messages from the Activity page.
+. Once a message flow is successfully deployed, you can track its processed messages from the Activity page.
+
+== Test Message Flows
+
+There are a couple of reasons why you should send test messages through the message flow after you deploy:
+
+* Ensure that you are receiving messages and any acknowledgments from your partners.
+
+* Make sure that the DataWeave map is working and your backend systems receive the transformed message.
```

Comment:
```suggestion
* The DataWeave map is working as intended and your backend systems transformed messages.
```

Reply:
WE need recieved in there because the backend systems don't transform. Gonna leave as is.